PWGCF/FemtoDream/FemtoDreamV0Selection.h

```diff
@@ -227,7 +227,7 @@ std::array<cutContainerType, 5> FemtoDreamV0Selection::getCutContainer(C const&
   const auto cpav0 = v0.v0cosPA(col.posX(), col.posY(), col.posZ());
   const std::vector<float> decVtx = {v0.x(), v0.y(), v0.z()};
 
-  float observable;
+  float observable{};
```

Comment:
The same as above

Reply:
Ok, I can change it but the results is the same.